mpq-exploder/src/main/java/ninja/eivind/mpq/MpqArchive.java

```diff
@@ -39,9 +40,8 @@ public MpqArchive(String fileName) {
 
     public Map<String, ByteBuffer> getFiles(String... filesToExtract) throws IOException {
         Map<String, ByteBuffer> map = new HashMap<>();
-        try (FileInputStream inputStream = new FileInputStream(file)) {
-            FileChannel fileChannel = inputStream.getChannel();
-
+        try (FileInputStream inputStream = new FileInputStream(file);
+                FileChannel fileChannel = inputStream.getChannel();) {
```

Comment:
Same code is also used in `MPQArchiveTest.testGetHeader()`, but I didn't bother changing it there, because it's just a test.